library/include/rocblas-functions.h

```diff
@@ -3986,6 +3986,281 @@ ROCBLAS_EXPORT rocblas_status rocblas_ztrmv_strided_batched(rocblas_handle
                                                             rocblas_stride                stridex,
                                                             rocblas_int batch_count);
 
+/*! \brief BLAS Level 2 API
+
+    \details
+    tpmv performs one of the matrix-vector operations
+
+         x = A*x or x = A**T*x,
```

Comment:
Should probably mention that A is a packed matrix here.

Reply:
yes

Reply:
done.